### DISCUSSION TITLE
git remote doesn,t work

### DISCUSSION THREAD

**Discussion by Fatemehshirazi** [community/community#21929] [New to GitHub]
<p>Hi,</p>
<p>I have created a repository in GitHub and want now connect this with my local repo. But git remote doesn,t work.</p>
<p>Appreciate if somebody who knows what is the problem help me.</p>
<p>BR,</p>
<p>/Fatemeh</p>
<p>PureRumble@PUREMAINFRAME MINGW32 ~<br>
$ git remote add origin <a href="https://github.com/Fatemehshirazi/Shiraz.git" rel="nofollow noopener">https://github.com/Fatemehshirazi/Shiraz.git</a><br>
fatal: Not a git repository (or any of the parent directories): .git</p>

**Comment by mpboom** [comment]
<p>Did you </p>
<pre><code>cd
</code></pre>
<p>into your local repo?</p>

**Comment by Fatemehshirazi** [comment, marked as answer]
<p>Thanks for replying. Yes I was in local directory. I have solved the problem. There was something wrong im my command lines for remote and push. </p>

**Reply by WeirdCodingGoat** [reply]
What was wrong?
